React/Modules/RCTRedBox.m

```diff
@@ -169,6 +169,14 @@ - (void)copyStack
   
   UIPasteboard *pb = [UIPasteboard generalPasteboard];
   [pb setString:fullStackTrace];
+  
+#if TARGET_IPHONE_SIMULATOR
+  // if running in the simulator, try to get the stack up to the hosting computer's clipboard as
+  // well as the pastboard in the simulator
+  if (_actionDelegate && [_actionDelegate respondsToSelector:@selector(redBoxWindow:sendTextToPasteboard:)]) {
+    [_actionDelegate redBoxWindow:self sendTextToPasteboard:fullStackTrace];
+  }
+#endif
```

Comment:
That would involve passing a lot of internal state (or exposing it in the RCTRedBoxWindow) for the delegate to be able to access it to build up the stack info. As it is right now, we could send other text to the pasteboard that isn't the stack. I can definitely see moving the UIPasteboard part in to the delegate, though, so that CopyStack builds up the stack string and then the action delegate handles all of the pasteboard/clipboard duties.